README.md

```diff
@@ -11,74 +11,128 @@
         <img src="https://img.shields.io/powershellgallery/p/AsBuiltReport.NetApp.ONTAP.svg" /></a>
 </p>
 <p align="center">
-    <a href="https://github.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/graphs/commit-activity" alt="GitHub Last Commit">
-        <img src="https://img.shields.io/github/last-commit/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/master.svg" /></a>
-    <a href="https://raw.githubusercontent.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/master/LICENSE" alt="GitHub License">
-        <img src="https://img.shields.io/github/license/AsBuiltReport/AsBuiltReport.NetApp.ONTAP.svg" /></a>
-    <a href="https://github.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/graphs/contributors" alt="GitHub Contributors">
-        <img src="https://img.shields.io/github/contributors/AsBuiltReport/AsBuiltReport.NetApp.ONTAP.svg"/></a>
+    <a href="https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP/graphs/commit-activity" alt="GitHub Last Commit">
+        <img src="https://img.shields.io/github/last-commit/rebelinux/AsBuiltReport.NetApp.ONTAP/master.svg" /></a>
+    <a href="https://raw.githubusercontent.com/rebelinux/AsBuiltReport.NetApp.ONTAP/master/LICENSE" alt="GitHub License">
+        <img src="https://img.shields.io/github/license/rebelinux/AsBuiltReport.NetApp.ONTAP.svg" /></a>
+    <a href="https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP/graphs/contributors" alt="GitHub Contributors">
+        <img src="https://img.shields.io/github/contributors/rebelinux/AsBuiltReport.NetApp.ONTAP.svg"/></a>
 </p>
 <p align="center">
     <a href="https://twitter.com/AsBuiltReport" alt="Twitter">
             <img src="https://img.shields.io/twitter/follow/AsBuiltReport.svg?style=social"/></a>
 </p>
 
-# NetApp ONTAP As Built Report
+# NetApp Ontap Arrays AsBuiltReport
+
+NetApp Ontap AsBuiltReport is a module of the parent "AsBuiltReport" [project](https://github.com/AsBuiltReport/AsBuiltReport). AsBuiltReport is a PowerShell module which generates As-Built documentation for many common datacentre infrastructure systems. Reports can be generated in Text, XML, HTML and MS Word formats and can be presented with custom styling to align with your company/customer's brand.
+
+For detailed documentation around the whole project, please refer to the `README.md` file in the parent AsBuiltReport repository (linked to above). This README is specific only to the NetApp Ontap Array repository.
+
+## :books: Sample Reports
+
+### Sample Report - Custom Style 1
+
+Sample NetApp Ontap As Built report HTML file: [Sample NetApp As-Built Report.html](https://raw.githubusercontent.com/rebelinux/AsBuiltReport.NetApp.ONTAP/dev/Samples/Sample%20NetApp%20As-Built%20Report.html)
+
+Sample NetApp Ontap As Built report with health checks, using custom report style.
```

Comment:
ONTAP should be in uppercase in keeping with NetApp branding, and consistent throughout the docs.